### ISSUE TITLE
Majora's Mask camera delay

### ISSUE THREAD

**Issue by ichee** [gonetz/GLideN64#1406]
Taking a picture with the camera results in a 5 second delay before picture is taken. Occurs regardless of settings, including hle and lle. Didn't occur with glide64.

**Comment by gonetz** [comment]
I have similar delay with glide64. I admit that it is a problem though: camera makes shot almost instant with angrylion's plugin. I guess the problem is similar to subscreen delay in Zelda OOT, but only in this case even "copy white" hack does not remove the delay. I have no idea yet how to fix it.

**Comment by ichee** [comment]
You're right. Glide64 didn't have the issue the first time I ran it, but did on subsequent runs. I tested every plugin release from the past 12 years and Angrylion's is the only one not affected.

**Comment by olivieryuyu** [comment]
can i have a savestate pls?

**Comment by ichee** [comment]
Here you go.
[The Legend of Zelda - Majora's Mask (U).pj.zip](https://github.com/gonetz/GLideN64/files/1271978/The.Legend.of.Zelda.-.Majora.s.Mask.U.pj.zip)

**Comment by olivieryuyu** [comment]
thanks

interesting to see that the ucode change exactly at this point from F3DZEX to S2DEX. I wonder why.

**Comment by IronYoshi64** [comment]
I know this is an old issue, but I found a workaround to pretty much fix this to some extent. For pretty much any video plugin, and you use 32bit Project64 (any dev builds that have the overclocking modifier option), set the Overclocking modifier to either 2-4, and set the Counter factor to 1. This also pretty much makes the game run smoother and more responsive then having it set to 2 with no overclocking. :)

**Comment by olivieryuyu** [comment]
the color is not white but purple i believe to get the matter fixed with a hack

**Comment by Jj0YzL5nvJ** [comment]
Try:
```
EnableCopyAuxiliaryToRDRAM = True
EnableN64DepthCompare = 2
```

[My last test is a bit old...](https://imgur.com/a/OXRdr2K)

**Comment by Jj0YzL5nvJ** [comment]
[`EnableCopyAuxiliaryToRDRAM = True` fix the delay, `EnableN64DepthCompare = 2` fixes the shadows...](https://imgur.com/a/Zc9jCYk)
